.gitattributes

```diff
@@ -5,9 +5,5 @@
 packages/**/*.d.ts -linguist-vendored
 # These icons are imported from Google.
 # Kept so that we can review the impact when changing the generation script.
-packages/material-ui-icons/lib linguist-generated
-packages/material-ui-icons/material-icons linguist-vendored
```

Comment:
This path still exists and still contains vendored svgs. Did you mean to convert this to a glob pattern as well?

Reply:
let's add them back for for readability.